CMakeLists.txt

```diff
@@ -1654,6 +1654,7 @@ if(ENGINEPRIME)
         -DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}
         -DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>
         -DCMAKE_PREFIX_PATH:PATH=${CMAKE_PREFIX_PATH}
+        -DCMAKE_INSTALL_LIBDIR:PATH=lib
```

Comment:
We use `-DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>` for libkeyfinder

Reply:
The Mixxx `CMakeLists.txt` file already specifies `CMAKE_INSTALL_PREFIX` for libdjinterop, so I'm not sure what you're suggesting?

The break was that the updated libdjinterop's `CMakeLists.txt` file would now install in `lib64` on Fedora, so this new line ensures that the install dir is always `lib`, which was the prior expectation.

Looks like keyfinder always installs in `lib`, regardless!

Reply:
We updated keyfinder upstream, but didn't update the commit hash here so we didn't run into this issue yet.

Reply:
Sorry, I got confused. I see you're already setting `-DCMAKE_INSTALL_PREFIX:PATH=<INSTALL_DIR>` here for libdjinterop.